doc/UsersGuide/source/IdealizedTC.rst

```diff
@@ -0,0 +1,82 @@
+.. role:: raw-html(raw)
+    :format: html
+
+.. _idealized-tc:
+
+**************************************
+Idealized Tropical Cyclone Case
+**************************************
+
+The idealized, regional tropical cyclone case is derived from the I-HAFS configuration (:cite:t:`Wang2024`) and is designed to support controlled studies of tropical cyclone dynamics and forecast development. This configuration removes real-world data assimilation and ocean coupling, focusing solely on atmospheric forecasts using idealized inputs.
+
+Initial and lateral boundary conditions (ICs/LBCs) are derived from a large-scale, idealized global FV3-based atmospheric forecast. The initial vortex is constructed using the Reed and Jablonowski (2011) method, introducing a weak, balanced storm into an environment favorable for rapid intensification. The preprocessing system generates fixed distributions of geography-related variables and constructs the ICs/LBCs from ``tcvitals`` and GRIB input files.
+
+The configuration mirrors the operational HAFS structure but simplifies terrain and surface properties. It includes:
+
+- Preprocessing to set up the forecast and nest domains  
+- Optional vortex initialization  
+- FV3-based forecast integration  
+- Postprocessing to generate GRIB2 and ATCF output files
+
+A utility called ``cal_vortex`` is available to recalculate wind, temperature, and humidity fields based on user-defined vortex specifications. In a recent experiment, altering damping settings resulted in a stronger, more compact vortex and a rightward track shift after 48 hours of forecast time.
```

Comment:
this is very specific to IHAFS and not included in the ufs-wm HSD case here; i would say remove this or make clear that this utility is available in IHAFS for users who wish to alter vortex specifications.